core/kazoo_ast/src/kapps_config_usage.erl

```diff
@@ -138,6 +142,24 @@ config_to_schema(Source, F='get_global', [_Account, Cat, K, Default], Schemas) -
         'undefined' -> Schemas;
         Key -> config_key_to_schema(Source, F, Document, Key, Default, Schemas)
     end;
+config_to_schema(Source, F='get_hierarchy', [Account, Cat, K], Schemas) ->
+    config_to_schema(Source, F, [Account, Cat, K, 'undefined'], Schemas);
+config_to_schema(Source, F='get_hierarchy', [_Account, Cat, K, Default], Schemas) ->
+    Document = category_to_document(Cat),
+    case key_to_key_path(K) of
+        'undefined' -> Schemas;
+        Key -> config_key_to_schema(Source, F, Document, Key, Default, Schemas)
+    end;
+config_to_schema(Source, F='get_with_strategy', [Strategy, Account, Cat, K], Schemas) ->
+    config_to_schema(Source, F, [Strategy, Account, Cat, K, 'undefined'], Schemas);
+config_to_schema(Source, F='get_with_strategy', [?BINARY_MATCH([?BINARY_STRING("hierarchy_merge")]), _Account, Cat, K, Default], Schemas) ->
+    Document = category_to_document(Cat),
+    case key_to_key_path(K) of
+        'undefined' -> Schemas;
+        Key -> config_key_to_schema(Source, F, Document, Key, Default, Schemas)
+    end;
+config_to_schema(_, 'get_with_strategy', _Args, Schemas) ->
```

Comment:
Shouldn't there be a catchall clause on the `/5` version of `get_with_strategy` that uses the `Default` value too?